next.config.js

```diff
@@ -37,6 +37,17 @@ module.exports = withTypescript({
     config.resolve.alias['react-dom$'] = 'react-dom/profiling';
     config.resolve.alias['scheduler/tracing'] = 'scheduler/tracing-profiling';
 
+    /**
+     * To avoid including classnames and clsx we alias it to clsx
+     * classnames is used by
+     * react-draggable
+     * react-select
+     * material-table
+     * notistack
+     * recharts
+     */
+    config.resolve.alias.classnames = 'clsx';
```

Comment:
Could you add `classnames` to `.yarnclean` (so that we can be sure not to bundle classnames) and check if this still works on landing and navigation? I suspect this doesn't work because of https://github.com/zeit/next.js/issues/7588

Reply:
Added, seems to work on `/components/snackbars/#notistack`

Reply:
For landing and navigation?

Reply:
And yeah webpack alias wont do unless the issue is fixed since we rely on landing to work on every page.

Reply:
Yeah

Reply:
Switched to using the babel config instead, solved the issues and still works as expected with `yarn start`

Reply:
Yeah I forgot that this only failed once we actually use ES6 modules for the client build.